pandas/tests/strings/test_cat.py

```diff
@@ -11,7 +11,13 @@
     _testing as tm,
     concat,
 )
-from pandas.tests.strings.test_strings import assert_series_or_index_equal
```

Comment:
this function was defined in another file, but only used here. so, just moving it here